### ISSUE TITLE
Npm task: Allow selection of feed view for internal registries

### ISSUE THREAD

**Issue by totkeks** [microsoft/azure-pipelines-tasks#8466]
When using Azure DevOps, I wanted to use views on the registry to distinguish in-development packages from stable or production ready packages. For this I want to use views (@release, @prerelease) to designate those packages.
Unfortunately the npm task in my pipeline does not allow me to select the view for the selected internal feed / registry.

Please make it possible to select the view, if one wishes to not use the default one.
As a nice to have feature, also disable the publish command, when a view is selected, though the task will fail anyway when run.

Workaround: manually pass the --registry parameter to the install command.

**Comment by elbatk** [comment]
Hello! I'm a PM over on the Azure Artifacts team. I wanted to drop in here and address some of these issues with the news that we're currently reevaluating the types of tasks we offer in Azure Pipelines, and looking to focus more on lighter-weight tasks that cover authentication and setup. In lieu of the task redesign, we're going to forego making changes to the current tasks (with a few exceptions). We hope the new tasks we spin up in the next quarter or so are easier and more helpful, be on the lookout!

Addition: it also looks like you have a workaround here, so I'm assuming you're not completely blocked by this issue, I hope!
